canonical-web-and-design/ubuntu.com
# Move /tco-caclulator to /openstack/tco-caclulator

## Done

- Move /tco-caclulator to /openstack/tco-caclulator
- Updated all links
- Added redirect
- Updated the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/tco-calculator
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to /openstack/tco-calculator and that it has the navigation of openstack on it

## Issue / Card

Fixes #9229
